ocp_resources/datavolume.py

```diff
@@ -272,26 +277,64 @@ def _check_none_pending_status(self, failure_timeout=120):
             self.logger.error(f"{self.name} status is {sample}")
             raise
 
-    def wait_for_dv_success(self, timeout=600, failure_timeout=120):
+    def wait_for_dv_success(
+        self,
+        timeout=TIMEOUT_10MINUTES,
+        failure_timeout=TIMEOUT_2MINUTES,
+        stop_status=None,
+        *args,
```

Comment:
```*args,
        **kwargs,
```
please rename to `*stop_status_args` and `**stop_status_kwargs`

and use them (not used right now)

Reply:
I'm not quite sure we need them at all. I expect that the passed function will do all the checks, and we don't know how this function will look like and what arguments should it get.

Reply:
Done, thanks @myakove, @rnetser for your help!